public/js/selfoss-base.js

```diff
@@ -75,7 +90,9 @@ var selfoss = {
             window.setInterval(selfoss.dbOnline.sync, 60*1000);
 
             window.setInterval(selfoss.ui.refreshEntryDatetimes, 60*1000);
-        });
+
+            selfoss.ui.showMainUi();
```

Comment:
The UI should be shown before the events are initialised otherwise events.resize will be fired prematurely and the custom scroll bar will be calculated incorrectly:

![](https://cloud.githubusercontent.com/assets/705123/26022220/542b3914-379f-11e7-8bc6-8b9cce57d3a4.png)